templates/wait-for-port.conf.erb

```diff
@@ -1,2 +1,2 @@
 [Service]
-ExecStartPost=/bin/bash -c 'while ! nc -z localhost <%= scope['qpid::ssl_port'] %>; do sleep 1; done'
+ExecStartPost=/bin/bash -c 'while ! ss --no-header --tcp --listening --numeric sport = :<%= scope['qpid::ssl_port'] %> | grep -q "^LISTEN.*:<%= scope['qpid::ssl_port'] %>"; do sleep 1; done'
```

Comment:
I guess we then should also adjust the manifest to install `ss`?

https://github.com/theforeman/puppet-qpid/blob/68f944ca15e7bd0d08ec5b06b430994a2e94c5e5/manifests/service.pp#L36-L39

Reply:
(it's in `iproute`)